src/framework/shortcuts/view/shortcutsmodel.h

```diff
@@ -51,15 +51,38 @@ class ShortcutsModel : public QAbstractListModel, public async::Asyncable
     Q_PROPERTY(QVariant currentShortcut READ currentShortcut NOTIFY selectionChanged)
 
 public:
+    inline static std::vector<TranslatableString> categories
+        = { TranslatableString("framework/actioncategory", "Undefined"),
+            TranslatableString("framework/actioncategory", "Internal"),
+            TranslatableString("framework/actioncategory", "Tablature"),
+            TranslatableString("framework/actioncategory", "Viewing & Navigation"),
+            TranslatableString("framework/actioncategory", "Playback"),
+            TranslatableString("framework/actioncategory", "Layout & Formatting"),
+            TranslatableString("framework/actioncategory", "Selecting & Editing"),
+            TranslatableString("framework/actioncategory", "Application"),
+            TranslatableString("framework/actioncategory", "Accessibility"),
+            TranslatableString("framework/actioncategory", "File"),
+            TranslatableString("framework/actioncategory", "Selection & Navigation"),
+            TranslatableString("framework/actioncategory", "Text & Lyrics"),
+            TranslatableString("framework/actioncategory", "Chord symbols & figured bass"),
+            TranslatableString("framework/actioncategory", "Measures"),
+            TranslatableString("framework/actioncategory", "Musical Symbols"),
+            TranslatableString("framework/actioncategory", "Dialogs & Panels"),
+            TranslatableString("framework/actioncategory", "Note Input"),
+            TranslatableString("framework/actioncategory", "Workspace"),
+            TranslatableString("framework/actioncategory", "Plugins") };
```

Comment:
Not sure if this is the best translation context. @shoogle What do you think? Maybe `"action/category"`, to keep it consistent with `"action"` which is the context of UIActions?
(probably "framework/" is not right anyway because it's not part of the framework module anymore)

Reply:
Please let me know if this is supposed to be done. Till then I'll push some of the other changes